apps/backend/lib/src/auth.ts

```diff
@@ -0,0 +1,77 @@
+import { Request, Response, NextFunction } from "express";
+import * as jwt from "jsonwebtoken";
+import { Algorithm } from "jsonwebtoken";
+
+interface UserToken {
+  userId: number;
+}
+
+export interface AuthenticatedRequest extends Request {
+  user: UserToken;
+}
+
+const SIGNOPTIONS: jwt.SignOptions = {
+  algorithm: "RS256" as Algorithm,
+  expiresIn: "14 days",
```

Comment:
I think we agreed on a shorter expiry, but this is probably much nicer for dev

Reply:
I don't recall discussing an expiration time, but I have no issues changing this in the future